juniper_codegen/src/derive_enum.rs

```diff
@@ -90,13 +93,24 @@ impl EnumVariantAttrs {
                     continue;
                 }
                 if let Some(AttributeValue::String(val)) =
-                    keyed_item_value(&item, "deprecated", AttributeValidation::String)
+                    keyed_item_value(&item, "deprecation", AttributeValidation::String)
```

Comment:
This should be `deprecated`, right? And if so, it looks like repeated code and can be removed, as the next block is doing `match keyed_item_value(&item, "deprecated", AttributeValidation::String`

Reply:
Partially yes.

This check appears to be unnecessary for backwards compatibility, however I believe the reason that the line is in the `EnumVariantAttrs` impl is because I was using the same code block to handle it for both that and `ObjectFieldAttrs`.

In the __current version__ (specifically on master), the attribute name is inconsistent between enum variants which use `deprecated` and object fields which use `deprecation`.

This change, makes both support the same set of attributes in a backwards-compatible way.
Happy to remove it if that is undesirable.

If the use of `deprecation` on object fields is itself considered deprecated, I can remove this one on EnumAttrs, and if desired can also remove the same one on ObjFieldAttrs.

Reply:
Ah, yeah. Sorry, forgot about that previous support.